spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe Api::V1::UsersController do
```

Comment:
Its fine, but also use the pagination [helpers](https://github.com/Wolox/wor-paginate#test-helpers) included in the gem.